Neos.Flow/Classes/Persistence/Doctrine/EntityManagerConfiguration.php

```diff
@@ -15,18 +15,21 @@
 use Doctrine\Common\EventManager;
 use Doctrine\Common\EventSubscriber;
 use Doctrine\DBAL\Connection;
-use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Exception as DbalException;
```

Comment:
In Neos.Flow/Classes/Command/DoctrineCommandController.php you used `DBALException` (and that would be b/c AFAIS)